src/utils/getPlaylistData.js

```diff
@@ -1,11 +1,12 @@
+// @ts-check
 export default async function getPlaylistData(newPlaylist) {
-  const baseUrl = 'https://itunes.apple.com/lookup?id='
+  const baseUrl = '/api/playlist/'
 
   let response = await Promise.all(
     newPlaylist.map(({ id }) => fetch(baseUrl + id).then(res => res.json()))
   )
 
-  let data = await response.map(({ results }) => results[0])
+  let data = response.map(({ results }) => results[0])
```

Comment:
Please try to find a better name than `data`.

Reply:
Also: `let`?

Reply:
At first i had an await there and wasn't sure about the order of promises. As a promise has a pending status i thought that i need a let, because he would first set it as pending and later reassign it with the promise result.